akka-typed/src/main/scala/akka/typed/ActorSystem.scala

```diff
@@ -155,3 +157,41 @@ object ActorSystem {
    */
   def wrap(untyped: a.ActorSystem): ActorSystem[Nothing] = new ActorSystemAdapter(untyped.asInstanceOf[a.ActorSystemImpl])
 }
+
+final class Settings(val config: Config, val untyped: a.ActorSystem.Settings, val name: String) {
```

Comment:
This should be evolvable into a complete set of settings of its own until 3.0, at which point we can then make the BC-breaking change of removing the `untyped` reference.